synapse/storage/__init__.py

```diff
@@ -266,6 +277,37 @@ def _count_users(txn):
 
         return self.runInteraction("count_users", _count_users)
 
+    def count_monthly_users(self):
+        """
+        Counts the number of users who used this homeserver in the last 30 days
+        This method should be refactored with count_daily_users - the only
+        reason not to is waiting on definition of mau
+        """
+        def _count_users(txn):
+            yesterday = int(self._clock.time_msec()) - (1000 * 60 * 60 * 24 * 30)
```

Comment:
That is not `yesterday` but `month_ago` or similar. Please rename to avoid confusion